packages/react-keytips/src/docs/Spec.md

```diff
@@ -0,0 +1,203 @@
+  <div role="tooltip" id="tooltip-2" class="{tooltip}" data-popper-placement="bottom">B</div>
+  <div role="tooltip" id="tooltip-3" class="{tooltip}" data-popper-placement="bottom">C</div>
+</div>
+```
+
+## Migration
+
+See [MIGRATION.md](./MIGRATION.md).
+
+## Behaviors
+
+## Entering and Exiting Keytips Mode
+
+- When the `startSequence` is pressed, the keytips component will enter keytips mode. By default the `startSequence` is `'alt+meta'` (option+control on macOS).
+  After entering keytips mode, the root Keytips will be visible (the ones that have no parent Keytip).
+
+- When the `returnSequence` is pressed, the parent keytips of current keytip will be shown, if it's the first (root) level of keytip, then the keytips mode will be exited.
+
+- When the `exitSequence` is pressed, the keytips mode will be exited. It's also possible to exit keytips mode by pressing the `startSequence` again, or by
+  mouse click outside of the keytips. It's also exiting from keytips mode on resize of the window or scroll.
+
+## Keytip activation
+
+- Keytip activation is done by pressing the key sequence that is defined in `keySequences` prop of `useKeytipRef` hook.
+  When the key sequence is pressed, the `onExecute` callback is triggered, if keytip has children keytips it will show them.
+
+- Keytip sequence is case-insensitive and limited to 3 characters.
+
+## Keytip on disabled item
+
+- Unlike Fabric, the `Keytip` won't appear for disabled target.
+
+## Keytip positioning
+
+Keytip component using [positioning API](https://react.fluentui.dev/?path=/docs/concepts-developer-positioning-components--docs) and
+can be controlled by `positioning` prop in `useKeytipRef` hook. By default the keytip is positioned below and centered to the target element.
+
+## Accessibility
+
+In terms of accessibility, `Keytip` component is very similar to `Tooltip` - it's a small popup that displays information related to an element, with difference, that `Keytip` can't be displayed by mouse hover over target element. Multiple keytips can be visible at the same time.
+
+- Each `Keytip` is assigned the role `"tooltip"`.
+- The `Keytip` target element references the corresponding keytip element using the `"aria-describedby"` attribute, providing a clear association between the target and the keytip (tooltip).
+- `Keytips` component has `content` prop, which is responsible for adding the `"aria-describedby"` for default start key sequence.
+- `Keytip` is not focusable.
+- `Keytip` adds `"aria-keyshortcuts"` attribute to the target element, which contains the key sequence that will trigger the keytip.
+- Unlike Tooltip, pressing `Escape` does not always hide the `Keytip`. By default, `Keytip` is configured to use `Escape` as a return sequence. Therefore, if there are ancestor keytips, pressing it will not hide the current `Keytip` but will instead display the ancestor keytips.
+- Focus stays on the trigger element while `Keytip` is displayed.
```

Comment:
@smhigley 👋🏻 

Here are a couple of a11y things I am not sure about: 
- Can `Keytip` have role `"tooltip"`?  (in the previous version there was no role for Keytip)
- In `v8` Keytip was adding a `aria-describedby` on the target element with id of the root layer and current keytip (so it's announcing start sequence and sequence of current keytip). 
-  In the current draft implementation I added both, so after the `aria-describedby` announced it will announce`shortcuts available`  of `aria-keyshortcuts`. So, question if we can have both? or if not, which way is better in your opinion? 

I'd be very grateful if you could take a look on this :)